applications/MeshingApplication/tests/parmmg_eulerian_test/parmmg_sphere_reference_mdpa_rank_1.mdpa

```diff
@@ -0,0 +1,180 @@
+Begin Properties 0
+
+End Properties
+
+Begin Properties 1
+    DENSITY  1.225
+    DYNAMIC_VISCOSITY  1.846e-05
+
+End Properties
+
+Begin Nodes
+	2	0.398333	-0.398333	0.503759
+	3	-1	1	-1
+	4	-1	1	2
+	5	1	1	2
+	6	1	1	-1
+	7	-1	-1	-1
+	8	1	-1	2
+	9	-1	-1	2
+End Nodes
+
+Begin Elements	Element3D4N
+	7	1	2	3	4	5	
+	8	1	6	3	7	2	
+	9	1	7	3	4	2	
+	10	1	8	2	5	6	
+	11	1	6	3	2	5	
+	12	1	8	2	9	5	
+End Elements
+
+Begin Conditions	SurfaceCondition3D3N
+	7	0	3	4	5	
+	8	0	6	7	3	
+	9	0	7	4	3	
+	10	0	8	6	5	
+	11	0	6	3	5	
+	12	0	8	5	9	
+End Conditions
+
+Begin NodalData	DISTANCE
+2	0	0
+3	0	0
+4	0	0
+5	0	0
+6	0	0
+7	0	0
+8	0	0
+9	0	0
+End NodalData
+
+Begin NodalData	DISTANCE_GRADIENT_X
+2	0	0
+3	0	0
+4	0	0
+5	0	0
+6	0	0
+7	0	0
+8	0	0
+9	0	0
+End NodalData
+
+Begin NodalData	DISTANCE_GRADIENT_Y
+2	0	0
+3	0	0
+4	0	0
+5	0	0
+6	0	0
+7	0	0
+8	0	0
+9	0	0
+End NodalData
+
+Begin NodalData	DISTANCE_GRADIENT_Z
+2	0	0
+3	0	0
+4	0	0
+5	0	0
+6	0	0
+7	0	0
+8	0	0
+9	0	0
+End NodalData
+
+Begin NodalData	PARTITION_INDEX
+2	0	1
+3	0	1
+4	0	1
+5	0	1
+6	0	1
+7	0	1
+8	0	1
+9	0	1
+End NodalData
+
+Begin SubModelPart	Outlet3D_Outlet_pressure_Auto1
+	Begin SubModelPartData
+	End SubModelPartData
+	Begin SubModelPartTables
+	End SubModelPartTables
+	Begin SubModelPartNodes
+		4
+		5
+		8
+		9
+	End SubModelPartNodes
+	Begin SubModelPartElements
+	End SubModelPartElements
+	Begin SubModelPartConditions
+		12
+	End SubModelPartConditions
+End SubModelPart	
+
+Begin SubModelPart	Parts_Parts_Fluid
+	Begin SubModelPartData
+	End SubModelPartData
+	Begin SubModelPartTables
+	End SubModelPartTables
+	Begin SubModelPartNodes
+		2
+		3
+		4
+		5
+		6
+		7
+		8
+		9
+	End SubModelPartNodes
+	Begin SubModelPartElements
+		7
+		8
+		9
+		10
+		11
+		12
+	End SubModelPartElements
+	Begin SubModelPartConditions
+	End SubModelPartConditions
+End SubModelPart	
+
+Begin SubModelPart	NoSlip3D_No_Slip_Auto1
+	Begin SubModelPartData
+	End SubModelPartData
+	Begin SubModelPartTables
+	End SubModelPartTables
+	Begin SubModelPartNodes
+		3
+		4
+		5
+		6
+		7
+		8
+		9
+	End SubModelPartNodes
+	Begin SubModelPartElements
+	End SubModelPartElements
+	Begin SubModelPartConditions
+		7
+		9
+		10
+		11
+	End SubModelPartConditions
+End SubModelPart	
+
+Begin SubModelPart	AutomaticInlet3D_Automatic_inlet_velocity_Auto4
+	Begin SubModelPartData
+	End SubModelPartData
+	Begin SubModelPartTables
+	End SubModelPartTables
+	Begin SubModelPartNodes
+		3
+		6
+		7
+	End SubModelPartNodes
+	Begin SubModelPartElements
+	End SubModelPartElements
+	Begin SubModelPartConditions
+		8
+	End SubModelPartConditions
+End SubModelPart	
+
```

Comment:
seems like you added the partitioned files by mistake

Reply:
these two mdpa files are meant to be the reference files to check that the mesh produced by parmmg has the correct number of submodelpars, elems, conds etc.

Reply:
ah ok
maybe change the name then to sth with "ref"? => would cause less confusion, I thought those were from the partitioner

In any case, why don't you rewrtie the test such that it reads the mdpas and checks the ModelParts? => then the test would also be independent from the number of ranks you use if I understand correctly

Reply:
> ah ok
> maybe change the name then to sth with "ref"? => would cause less confusion, I thought those were from the partitione
> 

The name does have "reference" in it :D: `parmmg_sphere_**reference**_mdpa_rank_1.mdpa
`
> In any case, why don't you rewrtie the test such that it reads the mdpas and checks the ModelParts? => then the test would also be independent from the number of ranks you use if I understand correctly

Then I would need to have as many ref files as rank possibilites no?

if mpi_size = 2 read: reference_mdpa_0 and reference_mdpa_1

if mpi_size = 3 read: reference_mdpa_0 and reference_mdpa_1 reference_mdpa_2, but 0 and 1 are different from the previous

if mpi_size = 4 read: reference_mdpa_0 and reference_mdpa_1 reference_mdpa_2 reference_mdpa_3, but 0,1,2 are different from the previous

Reply:
It can be done but I wanted to avoid storing so many mdpa files

Reply:
> The name does have "reference" in it :D: parmmg_sphere_**reference**_mdpa_rank_1.mdpa 

I am looking at too many things at the same time :facepalm: